Copy edits (#132)
websight-io/docs: docs/docs/developers/development/dialogs/container/index.md

## Description:

> The Container component allows the user to group multiple elements into a container. It is useful for showing or hiding a group of components.

Reviewer: ```suggestion
The Container dialog field allows the user to group multiple elements into a container. It is useful for showing or hiding a group of dialog fields.
```

> The Container component allows the user to group multiple elements into a container. It is useful for showing or hiding a group of components.

Reviewer 2: ```suggestion
The Container component allows the user to group multiple elements into a container. For example, it is useful for showing or hiding a group of components.
```

> The Container dialog field allows the user to group multiple elements into a container. It is useful for showing or hiding a group of dialog fields.

Reviewer 2: ```suggestion
The Container dialog field allows the user to group multiple elements into a container. For example, it is useful for showing or hiding a group of dialog fields.
```
